pipeline/plot/__init__.py

```diff
@@ -12,3 +12,10 @@
 #            group_psth, group_psth_ll]
 
 __all__ = [show_source]
+
+
+class PhotostimError(Exception):
```

Comment:
thinking we should move all error classes up to some toplevel file - currently:
```
$ grep -R 'class.*Error' pipeline
pipeline/ingest/__init__.py:class ProbeInsertionError(Exception):
pipeline/ingest/__init__.py:class ClusterMetricError(Exception):
pipeline/ingest/__init__.py:class BitCodeError(Exception):
pipeline/ingest/__init__.py:class IdenticalClusterResultError(Exception):
pipeline/ingest/histology.py:class HistologyFileError(Exception):
pipeline/histology.py:    class ElectrodePositionError(dj.Part):
```
but unrelated to this, so just a note for now